insights/parsers/pci_rport_target_disk_paths.py

```diff
@@ -0,0 +1,227 @@
+"""
+PCIRportTargetDiskPath
+======================
+
+Module for parsing the output of command ``find /sys/devices/pci0000:00 -mindepth 8 -maxdepth 8``.
+"""
+
+from insights.parsers import ParseException, SkipException
+from insights import parser, CommandParser
+from insights.specs import Specs
+
+
+@parser(Specs.pci_rport_target_disk_paths)
+class PCIRportTargetDiskPaths(CommandParser, dict):
+    """
+    Class for parsing ``find /sys/devices/pci0000:00 -mindepth 8 -maxdepth 8`` command output.
+
+    Typical output of command ``find /sys/devices/pci0000:00 -mindepth 8 -maxdepth 8`` with
+    the filter of 'block' looks like::
+
+        /sys/devices/pci0000:00/0000:00:01.0/0000:04:00.6/host1/rport-1:0-1/target1:0:0/1:0:0:0/block/sdb
+        /sys/devices/pci0000:00/0000:00:01.0/0000:04:00.7/host2/rport-2:0-2/target2:0:0/2:0:0:0/block/sdc
+        /sys/devices/pci0000:00/0000:00:02.2/0000:02:00.0/host0/target0:1:0/0:1:0:0/block/sda/dev
+        /sys/devices/pci0000:00/0000:00:02.2/0000:02:00.0/host0/target0:1:0/0:1:0:0/block/sda/sda1
+        /sys/devices/pci0000:00/0000:00:02.2/0000:02:00.0/host0/target0:1:0/0:1:0:0/block/sda/sda2
+
+    The Original data parsed looks like::
+
+            {
+                'PCI': [
```

Comment:
If there is ONLY one `key` in the `dict`, why not just set it as a `list` instead of `dict`?